articles/native-platforms/ios-objc/08-calling-APIs.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Calling your API
+description: This tutorial will show you how to use Auth0 to authenticate with your own API server.
+---
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* CocoaPods 1.0.0
+* XCode 7.3 (7D175)
+* Simulator - iPhone 6 - iOS 9.3 (13E230)
+  :::
+
+<%= include('../../_includes/_package', {
+  pkgRepo: 'auth0-samples/auth0-ios-objc-sample',
+  pkgBranch: 'master',
+  pkgPath: '08-Calling-APIs',
+  pkgFilePath: '08-Calling-APIs/Auth0Sample/Info.plist',
+  pkgType: 'replace'
+}) %>
+
```

Comment:
@sebacancinos We need an intro text here. This is the text from react doc:
```
The reason for implementing authentication in the first place is to protect information. In this case your information is a resource served from a server of any sort. Auth0 provides a squad of tools to assist you with end-to-end authentication in an application. Auth0 suggests you conform to RFC standard by sending the token through Authorization header.
```
I think you should combine this with the text you have below (and remove the quote).